index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <section class="container">
+      <section class="header">
+        <nav>
+          <h3 class="mylogo">My Logo</h3>
+          <div class="menu-bars"></div>
+          <span class="material-symbols-outlined"> menu </span>
+        </nav>
+      </section>
+      <section class="main-content">
+        <h1>
+          I'm Anwaar<br />
+          Glad to see <br />
+          you!
+        </h1>
+        <p>
+          I'm a software developer! I can help you <br />
+          build a product, feature or website Look<br />
+          through some of my work and<br />
+          experience! If you like what you see and<br />
+          have a project you need coded, don't <br />
+          hesitate to contact me.
+        </p>
+        <a href="https://github.com/Anwaar12" class="connect">LET'S CONNECT</a>
+        <ul class="media-icons">
+          <li>
+            <img src="./images/twitter.svg" alt="twitter" />
+          </li>
+          <li>
+            <img src="./images/linkedin1.png" alt="linkedin" />
+            <img src="./images/linkedin2.png" alt="linkedin" />
+          </li>
+          <li><img src="./images/gmail.svg" alt="gmail" /></li>
+          <li>
+            <img src="./images/github.svg" alt="github" />
+          </li>
+          <li>
+            <img src="./images/Vector (3).svg" alt="peace" />
+          </li>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag), the `a` tag will let you connect your social networks with the icons, using href 😊 😉